examples/BUILD

```diff
@@ -88,6 +95,13 @@ jsonnet_to_json_test(
     name = "files_test",
     size = "small",
     src = "files.jsonnet",
+    ext_strs = {
```

Comment:
These changes appear unrelated to stamping?

Reply:
removed unrelated tests.